pkg/operator/controller/status.go

```diff
@@ -128,58 +129,77 @@ func computeStatusConditions(conditions []configv1.ClusterOperatorStatusConditio
 		degradedCondition.Message = "DNS namespace does not exist"
 	} else {
 		degradedCondition.Status = configv1.ConditionFalse
+		degradedCondition.Reason = "NamespaceExists"
```

Comment:
It looks like most operators use "AsExpected" for the `Degraded` condition's `Reason` field when its `Status` field is false:
```console
% oc get clusteroperators -o json | jq -r '.items|.[].status.conditions|.[]|select(.type=="Degraded")|[.reason][0]' | sort | uniq -c | sort -n
      1 NoCredentialsFailing
      5 null
     10 AsExpected
```
I'm not one to advocate blind conformity in all things, but I submit the above for your consideration.

Reply:
I'll use `AsExpected`.